src/utils/components/SSHSecretModal/hooks/useSecretsSaveBtnDisabled.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from 'react';
```

Comment:
It seems like this is a hook because of useMemo. Do we really need it, or can it be a regular function?

Reply:
I'll check.

Reply:
Done!